sdk-docs/getting-started/creating-a-new-extension.md

```diff
@@ -21,7 +21,7 @@ nx g @trackunit/iris-app:extend [name-of-your-extension]
 ### 2. Answer the following questions:
 
 - **What subdir would you like to use for this app-extension?:** 
-  - Trackunit recommends using the Feature (my-feature-1) directory to sort your extensions.
+  - Trackunit recommends bundling extensions as related yet independent components of larger features. You can decide to leave this field empty, and then your extension's folder will be created in the root. But if you are working on bigger things, you have the option to put extensions in the same subfolder. Functionally it doesn't have an impact, but helps structure your folders better.
```

Comment:
I don't really like the last sentence. Perhaps "I will not impact functionality, but it helps structuring your code."

Reply:
We could also link to resources on how to structure a mono-repo, since this is where the inspiration is drawn from.